Doc/using/cmdline.rst

```diff
@@ -994,3 +1000,110 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory that contains the
+input script, if there is one. Otherwise, the first entry is the current
+directory, which is the case when executing the interactive shell, a :ref:`-c <using-on-interface-option-c>`
+command, or :ref:`-m <using-on-interface-option-m>` module.
```

Comment:
Not in isolated mode. See the start of my PR adding -P option: https://github.com/python/cpython/pull/31542/files

```

   .. c:member:: int add_path0
      If non-zero, ``Py_RunMain()`` adds ``sys.path[0]``, it prepends the
      following directory to ``sys.path``:
      * The current working directory if :c:member:`argv[0] <PyConfig.argv>`
        is equal to ``L"-m"``(``python -m module``).
      * An empty string if :c:member:`argv[0] <PyConfig.argv>` is equal to
        ``L"-c"`` (``python -c code``).
      * :c:member:`argv[0] <PyConfig.argv>` otherwise (``python script.py``).
        Resolve symbolic links in this case.
      Set to 0 by the :option:`-P` command line option.
      Default: ``1`` in Python config, ``0`` in isolated config.
      .. versionadded:: 3.11
```